server/models/UserModel/index.js

```diff
@@ -2,44 +2,28 @@ import { collections } from '../../utils/mongoCollections'
 
 const userMongoProjection = {
   _id: 0,
-  uid: 1,
-  display_name: 1,
-  acl: 1,
-  role: 1,
-  icon: 1,
-  mail: 1,
-  access: 1,
-  account_expires: 1,
+  password: 0,
+  bad_pwd_count: 0,
+  lockout_time: 0,
+  last_logoff: 0,
+  last_logon: 0,
+  force_reset_pw: 0,
 }
 
 const UserModel = {
-  withDefaults: (overrides = {}) => ({
-    display_name: '',
-    title: '',
-    department: '',
-    company: '',
-    mail: '',
-    member_of: '',
-    bad_pwd_count: 0,
-    lockout_time: 0,
-    last_logoff: Date.now(),
-    last_logon: Date.now(),
-    account_expires: null,
-    password: '',
-    ldap: false,
-    force_reset_pw: false,
-    realms: [''],
-    icon: '',
-    access: [],
-    blocked: false,
-    role: 'member',
-    ...overrides,
-  }),
   save: async (db, userAttributes) => {
     const newUser = UserModel.withDefaults(userAttributes)
 
     return await db.collection(collections.users).insertOne(newUser)
   },
+  show: async (db, uid) => {
```

Comment:
```suggestion
  find: async (db, uid) => {
```

Reply:
same as a above, find is already a mongo db method and it would confuse mongo users